jlm/llvm/opt/alias-analyses/Andersen.cpp

```diff
@@ -430,7 +430,7 @@ Andersen::AnalyzePhi(const phi::node & phi)
 
     // Map the output register to the recursion result's pointer object
     auto & outputRegister = *rv;
-    Set_->MapRegisterToExistingPointerObject(outputRegister, argumentRegisterPO);
+    Set_->MapRegisterToExistingPointerObject(outputRegister, resultRegisterPO);
```

Comment:
This modification would not affect the PointsToGraph previously, but now it actually makes a difference if the phi node's output is mapped to the result or the argument. (As you can see, the comment was previously a lie, now the comment is true)